RadixWallet/Profile/Entity/Account.swift

```diff
@@ -65,6 +65,10 @@ extension Profile.Network {
 		/// A required non empty display name, used by presentation layer and sent to Dapps when requested.
 		public var displayName: NonEmpty<String>
 
+		/// Flags that are currently set on entity.
+		@DefaultCodable.EmptyCollection
+		public var flags: Set<EntityFlag>
```

Comment:
Set will be painful once we add more Flags, since JSOn testing will be broken since order is unstable. Can we use OrderedSet? Or do you wanna defer the pain? (Non-deterministic random JSON encoding unit test failures)

Reply:
I did update to IdentifiedArray in the mean time